.infra/index.ts

```diff
@@ -394,7 +394,7 @@ if (isAdhocEnv) {
       maxReplicas: 2,
       limits: { memory: '1Gi' },
       requests: {
-        cpu: '1',
+        cpu: '200m',
```

Comment:
I remember digest needed more cpu, does it not use it from metrics?

Reply:
https://console.cloud.google.com/monitoring/dashboards/builder/4fd02e51-b395-43c8-97dc-852055e0ab30;duration=P14D?f.umlabel.app.App=api-personalized-digest&f.rlabel.namespace_name.Namespace=&project=devkit-prod

Reply:
👍